stack/base/before-notebook.d/10_prepare-home-rc.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# If the container is start by spawner and the home is remounted.
+# The .bashrc in HOME won't be set properly.
+if [[ ! -f  /home/${NB_USER}/.bashrc ]]; then
+    cat /etc/skel/.bashrc > /home/${NB_USER}/.bashrc
```

Comment:
Why not 
```suggestion
    cp /etc/skel/.bashrc > /home/${NB_USER}/.bashrc
```

(and same below). Seems weird to use `cat` for copying a file.

Reply:
If you use `cp`, you need to remove the `>` sign